### ISSUE TITLE
Cannot build with Java 16

### ISSUE THREAD

**Issue by gbreuil** [ATLauncher/ATLauncher#494]
**Describe the bug**
Build fail when i want to update the package atlauncher

**To Reproduce**
Build the package with pacmac app manager

**Expected behavior**
Update the software

**Logs**
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'ATLauncher'.
> Could not open cp_proj generic class cache for build file '/var/tmp/pamac-build-*username/atlauncher/src/ATLauncher-3.4.5.0/build.gradle' (/home/*username/.gradle/caches/6.7.1/scripts/1bm8qxzz39ld6h5x0q9sqcqcv).
   > BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 60
```

**System information (please complete the following information, or remove if not applicable):**

- OS (including distro/version): Arch Manjaro Linux 5.12.9
- Java Version: 16 (Not openJDK)

**Comment by RyanTheAllmighty** [comment]
Yeah it can't be built with Java 16 at this point in time as we need to update Gradle which cannot be done due to some issues with changes between the gradle versions.

**Comment by gbreuil** [comment]
So i need to downgrade my java os version just to build it or i can use a much simpler command ?

**Comment by simone-viozzi** [comment]
same issue, what is the latest supported version?

**Comment by simone-viozzi** [comment]
i had the same issue, needed to switch version of java with `sudo archlinux-java set java-8-openjdk` and install it. after the update you can re-switch back to java 16 with the same command.

**Comment by RyanTheAllmighty** [comment]
Yeah, it's been known for a while, but I just haven't had the time to properly sit down and do it as it requires a bit of rework and switching a few libraries around.

But yeah for now use a lower Java version or alternatively use our [atlauncher-bin](https://aur.archlinux.org/packages/atlauncher-bin) package which we build and push out as part of our release.

**Comment by RyanTheAllmighty** [comment]
Fixed with version 3.4.6.0
